tests/test_api_tcp_client.py

```diff
@@ -49,10 +49,13 @@
 
 device_type_api1 = DeviceType.TOUCH
 device_type_api2 = DeviceType.RUNNER
-device_index = 2
+device_type_token_api2 = DeviceType.RUNNER_S11
+device_index = 0
```

Comment:
Please add a test for index != 0 since the packet is different for index which is not 0

Reply:
It is done.